modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -1,15 +1,14 @@
-
 import * as path from 'path';
 import * as childProcess from 'child_process';
 import * as os from 'os';
 import * as WebdriverIO from 'webdriverio';
+import {RemoteOptions} from 'webdriverio';
```

Comment:
Formatting.

Reply:
One day we'll enable eslint on the project 😂